### PR TITLE
wsa-pacman: Add version 1.1.1

### PR DESCRIPTION
**Pull request by wenmin92**
[wsa_pacman](https://github.com/alesimula/wsa_pacman) is a GUI package manager and package installer for Windows Subsystem for Android (WSA).

Note:
I used the  portable package but did not find the configuration file. @alesimula, can you help with this question?

Relates to https://github.com/alesimula/wsa_pacman/issues/10

### CODE REVIEW

**Comment by rashil2000** [comment]
`"license": "GPL-3.0-only"`

This only works on 64bit OS, right?

**Comment by wenmin92** [comment]
> This only works on 64bit OS, right?

Thanks, while win11 only works on 64bit, this app should do so.

**Comment by wenmin92** [comment]
@rashil2000 I have added a depend and file associations scripts, please review it again. thx.

File: bucket/wsa-pacman.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1.1.1",
+    "description": "A GUI package manager and package installer for Windows Subsystem for Android (WSA)",
+    "homepage": "https://github.com/alesimula/wsa_pacman",
+    "license": "GPL-3.0-only",
+    "notes": "For file associations, run '$dir\\install-associations.reg'",
```
Comment (rashil2000) [inline bucket/wsa-pacman.json:6]:
```suggestion
    "notes": "For .apk file associations, run '$dir\\install-associations.reg'",
```

File: bucket/wsa-pacman.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "depends": "vcredist2019",
```
Comment (rashil2000) [inline bucket/wsa-pacman.json:7]: Is there any documentation for this? The usual practice is to use `suggest` instead of `depends` for vcredist's.

**Review by rashil2000 — CHANGES_REQUESTED** [review]
What about config/persistence?

File: bucket/wsa-pacman.json (hunk 2 above)
Comment (wenmin92) [reply bucket/wsa-pacman.json:7]: https://github.com/alesimula/wsa_pacman/issues/10#issuecomment-1000503332

Comment (rashil2000) [reply bucket/wsa-pacman.json:7]: I'd recommend using 'suggest' field.

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**